backend/appwrite-functions/announcement/README.md

```diff
@@ -0,0 +1,102 @@
+# General
+This function brings add and get announcements features to appwrite.
+- Client can send list of announcements to Backend. These data will be written to DB along with `created_time`
+- Client can request annoucements with some filters. The function will then query data from DB and print out to `stdout`.
+    Client can now request status of execution and retrieve data from the `stdout` or `stderr` field. See `backend/appwrite-functions/announcement/test.js`.
+- `Announcements` collection needed for the operations. Create with 
+
+    `python3 ../../database-collection-schemas/createAnnouncementCollection.py`.
+## Get announcements
+A simple function that parses request and return announcements (from admin) in string.
+
+A json payload looks like this:
+```json
+{ "data": "{\"getAnnouncements\":true,\"addAnnouncements\": false,\"numberOfAnnouncements\":3,\"timestamp\":1637100904,\"after\":true}" }
+```
+
+- The payload must contain `"data"` key with value in string. This string will be parsed to JSON by the function later on.
+- `getAnnouncements` must be set to true and `addAnnouncements` to false. Otherwise, function will ignore this request.
+- If `after` is true, the function will return #`numberOfAnnouncements` most recent announcements after the `timestamp`.
+
+## Add announcements
+
+```json
+{
+    "data": "{\"getAnnouncements\":false,\"addAnnouncements\": true, \"announcements\": [\"Message 3 added using HTTP request\", \"Message 4 added using HTTP request\"]}"
+}
+```
+- `getAnnouncements` must be set to false and `addAnnouncements` to true.
```

Comment:
Why are two different fields required for this? It increases error capability when they have incompatible settings.

Reply:
this is obsoleted. Changing...